hibernate-core/src/main/java/org/hibernate/dialect/H2Dialect.java

```diff
@@ -423,6 +423,11 @@ public boolean requiresParensForTupleDistinctCounts() {
 		return true;
 	}
 
+	@Override
+	public String toBooleanValueString(boolean bool) {
+		return bool ? "true" : "false";
```

Comment:
So I checked other dialect class implementing this method. It seems the following pattern is followed invariably:
```
return String.valueOf( bool );
```
so maybe we might do that as well?

Reply:
I actually copied this from the PostgreSQL81Dialect(https://github.com/hibernate/hibernate-orm/blob/e896db2ce2cdcaf2ed08e7333343b79cc228dc59/hibernate-core/src/main/java/org/hibernate/dialect/PostgreSQL81Dialect.java#L410), so it's not consistent in the code base. I don't see any advantage of using one over the other to be honest.

Reply:
okay, if so, that is fine.